bftclient/test/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ target_link_libraries(bft_client_test PRIVATE
   bftclient_new
   threshsign
   secretsmanager
+  stdc++fs
```

Comment:
https://releases.llvm.org/9.0.0/projects/libcxx/docs/UsingLibcxx.html#using-filesystem

"Starting with LLVM 9.0, support for <filesystem> is provided in the main library and nothing special is required to use <filesystem>."
Why is this needed?

Reply:
what about gcc?

Reply:
what tests did you do to make sure there is no degradation and everything is still working?

Reply:
what kind of degradation? We already use std::filesystem in different modules, also it's not used in a performance critical paths, but rarely, mainly during startup.

Reply:
I ment functionality degradation. making sure everything is still working.

Reply:
> what about gcc?
From https://gcc.gnu.org/gcc-9/changes.html:

Runtime Library (libstdc++)
Improved support for C++17, including:
The C++17 implementation is no longer experimental.
Parallel algorithms and <execution> (requires [Thread Building Blocks](https://github.com/oneapi-src/oneTBB) 2018 or newer).
<memory_resource>.
Using the types and functions in <filesystem> does not require linking with -lstdc++fs now.

What version of gcc is supposed to be supported?
If this is the case, it should be documented in the header.

Reply:
gcc won't compile without experimental and without -stdc++fs linkage.
right now I'm not changing the existing include/linkage behavior